src/stack_and_queue/stack/sorting_station.c

```diff
@@ -0,0 +1,95 @@
+#include "stack.h"
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+bool isOperatorFirstPriority(char c)
+{
+    return c == '*' || c == '/';
+}
+
+bool isOperatorSecondPriority(char c)
+{
+    return c == '+' || c == '-';
+}
+
+// Takes infix form of expression as string and returns pointer on postfix form string.
+char* sortingStation(char* infixForm)
+{
+    char* output = calloc(strlen(infixForm), sizeof(output) + 1);
+    Stack* operators = newStack();
+
+    int indexOfOutput = 0;
+    for (int indexOfInfix = 0; infixForm[indexOfInfix] != '\0'; ++indexOfInfix) {
+        if (isdigit(infixForm[indexOfInfix])) {
+            output[indexOfOutput] = infixForm[indexOfInfix];
+            output[indexOfOutput + 1] = ' ';
+            indexOfOutput += 2;
```

Comment:
```suggestion
            output[indexOfOutput++] = infixForm[indexOfInfix];
            output[indexOfOutput++] = ' ';
```
но это косметика, выбирайте, как нравится